From 0d02b002ed5679f9fda644c96c5f78b1bfafc939 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Thu, 1 Sep 2016 17:55:04 +0300
Subject: [PATCH 0905/2241] fix: dts: a70x0: changed eth1 speed from 2.5G to 1G

This port configuration was set to 2.5G for the Peridot switch.
since the board is released without the switch, the default
configuration is set to 1G.

Change-Id: Ib610485c3e2a7de108a8c0cb0689ab80bdbf2f00
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32483
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db-router.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
index 6e7da6c..844e689 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-router.dts
@@ -120,7 +120,6 @@
 				};
 				emac2: mac2 {
 					phy-mode = "sgmii"; /* lane-0 */
-					phy-speed = <2500>;
 				};
 				emac3: mac3 {
 					phy-mode = "rgmii"; /* rgmii-1 */
-- 
2.7.4

